### DISCUSSION TITLE
Solver Knitro could not be found

### DISCUSSION THREAD

**Discussion by NicoBravo** [yalmip/YALMIP#1210] [General discussion]
Hello,
I am trying to solve a NLP and I wanna use solver Knitro.
The problem is yalmip could not find knitro. I have added the solver folder knitromatlab whichi includes the necessary files, and also the license. But anytime I try to set up it as the solver, yalmip will tell the error code -3.
I run yalmiptest and only knitro could not be found, like this:
![](https://user-images.githubusercontent.com/85178112/212272302-31a32bec-7c31-43e7-ad36-3397a0898c93.png)

Thus I truly do not know why and I hope you could help me.
Thanks

**Comment by johanlofberg** [comment]
They have changed the file structure (and call interface) of Knitro, and I have not had the time to update YALMIP for the new version.
